web3.js/src/connection.js

```diff
@@ -1857,6 +1890,35 @@ export class Connection {
       });
   }
 
+  /**
+   * Returns epoch activation information for a stake account that has been delegated
+   */
+  async getStakeActivation(
+    publicKey: PublicKey,
+    commitment: ?Commitment,
+    epoch: ?number,
+  ): Promise<StakeActivationData> {
+    const args = this._buildArgs([publicKey.toBase58()], commitment);
+    if (epoch !== undefined) {
+      // options
+      args[args.length - 1].epoch = epoch;
```

Comment:
You'll need to make sure that the config object was actually added to `args` here. It's not added if both `commitment` is undefined and `connection.commitment` is undefined.

Reply:
done, I modified _buildArgs function.